core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/BaseResourceSearchRequest.java

```diff
@@ -98,6 +103,21 @@ protected final Expression prepareQuery(RepositoryContext context) {
 				.build());
 		}
 
+		if (containsKey(OptionKey.PROPERTIES)) {
+			final Collection<String> properties = getCollection(OptionKey.PROPERTIES, String.class);
```

Comment:
A few example test cases would be nice to verify the functionality. Also, not sure if dot is the best separator for key and value pairs. What happens with nested settings, like `settings.nested.prop.field`.